### DISCUSSION TITLE
how to join (dev)

### DISCUSSION THREAD

**Discussion by spazestudios** [NovaoTech/solaris#19] [Q&A] [answered]
I would like to join (im mildly experienced in html css and python) and have made a website before :) I would LOVE to help with this but cant figure out how to join.

**Comment by icarusinthewings** [comment, marked as answer]
> I would like to join (im mildly experienced in html css and python) and have made a website before :) I would LOVE to help with this but cant figure out how to join.

Hey @Spazestudios!
If you'd like to join, just take a look at our projects page for this repository. If you see something you think you can do, just fork, modify, and pull request. I'll take a look at it and then merge it if it is good.
Thanks for your interest in our project!
~ Velocity

**Reply by spazestudios** [reply]
alright! thx for helping !
